torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -257,7 +258,17 @@ class ProcessGroup : public torch::CustomClassHolder {
   virtual c10::intrusive_ptr<ProcessGroup::Work> reduce_scatter(
       std::vector<at::Tensor>& outputTensors,
       std::vector<std::vector<at::Tensor>>& inputTensors,
-      const ReduceScatterOptions& opts = ReduceScatterOptions()) = 0;
+      const ReduceScatterOptions& = ReduceScatterOptions()) = 0;
+
+  virtual c10::intrusive_ptr<ProcessGroup::Work> _reduce_scatter_base(
+      at::Tensor&,
+      at::Tensor&,
+      const ReduceScatterOptions& opts = ReduceScatterOptions())
+      {
+        // does not support it by default
+        throw std::runtime_error("ProcessGroup does not support alltoall");
```

Comment:
nit: "ProcessGroup does not support _reduce_scatter_base"

Reply:
Thanks!